src/test/java/com/whiteclarkegroup/liquibaselinter/integration/IgnoreFilesPatternIntegrationTest.java

```diff
@@ -29,6 +29,17 @@ void registerTests() {
             "Should ignore file with no comment xml nested",
             "ignore-files-pattern/nested/root-change-log.xml",
             "ignore-files-pattern/lqlint-ignore-nested.json");
+
+        shouldPass(
+            "Should ignore file with spaces when in ignore pattern",
+            "ignore-files-pattern/nested/ignore/ignore change log with spaces.xml",
```

Comment:
I think the intent of this test is a little unclear, since the example file names suggest it is about the spaces in the names, but the actual rule we are testing against the the has-comment one (which I think is fine).